docs/src/understand/digital-twin.md

```diff
@@ -0,0 +1,170 @@
+  from which data is collected and to which operation requests are forwarded.
+
+It's important to note that a digital twin is a user perspective
+that might differ from the physical architecture of the device.
+Indeed, the hardware components might have specificities irrelevant for the operators,
+and these technical details are abstracted away by the digital twin.
+
+## Building Blocks
+
+Thin-edge represents a piece of equipment as a hierarchy of devices and services,
+with a main device, nested child-devices and the services running on these devices.
+
+### Device
+
+A device represents a hardware component which firmware, software packages and configuration files are managed from the cloud.
+This is also a target of maintenance operations and a source of telemetry data.
+
+A device needs not always be a physical device, but can also be a logical one,
+abstracting some parts or groups of parts of the equipment.
+
+### Service
+
+A service represents a piece of software running on a device and that is monitored from the cloud.
+It can be an application, an operating-system process, an industrial process or a thin-edge daemon.
+
+From a digital twin perspective, a service is essentially a source of monitoring data;
+but could also be a target of maintenance operations such as configuration updates and restarts.
+
+### Main Device
+
+The main device represents the piece of equipment as a whole.
+This is usually, but not necessarily, the gateway device where thin-edge is running.
+
+### Child Device
+
+A child device is a device that is attached to the equipment.
+A child device can have further nested child devices as well,
+forming a hierarchy, with the main device as root and the services as leafs,
+zooming in from the piece of equipment into its parts.
+
+### Device Profile
+
+A device profile groups what defines the behavior of a device:
+its __firmware__, __software packages__ and __configuration files__.
+
+The actual profile of a device is represented in its digital twin
+from where it can be managed, checked and updated.
+
+There are no strict definitions of the different kind of profile parts.
+For instance, some AI model can be seen as a software package or a configuration files.
+The differences are more related to tooling and update constraints.
+The firmware cannot be removed and might require a device restart on update.
+A configuration file can be edited by an operator,
+while a software package usually downloaded from a repository.
+
+### Telemetry Data
+
+Telemetry data encompass __measurements__, __events__ and __alarms__
+that are collected, generated or raised by sensors and processes.
+
+These data are propagated by thin-edge from the physical devices to their digital twins;
+and made available to the cloud application, users and operators.
+
+Note that, the term "telemetry data" is also used to
+represent metrics and monitoring data related to the operating system and running processes.
```

Comment:
An admonish might be a better fit (though only a suggestion...)

```suggestion
:::note
The term "telemetry data" is also used to
represent metrics and monitoring data related to the operating system and running processes.
:::
```

Reply:
fixed